action.yml

```diff
@@ -33,6 +33,10 @@ inputs:
     description: 'Bash script to be run after make test inside the build folder'
     required: false
     default: ''
+  gzdev_project_name:
```

Comment:
I think you could get the project name from the github context, something like this:

`${{ github.repository }}`

This should return the owner / repo name, such as `ignitionrobotics/ign-transport`.

This means we don't need to go back and change every branch of every repository to enable this, which is going to be dozens of PRs if we need to.

Reply:
> I think you could get the project name from the github context, something like this:
> 
> `${{ github.repository }}`
> 
> This should return the owner / repo name, such as `ignitionrobotics/ign-transport`.

Thought on sniffing the repo name to infer the name but I think that it would rest some flexibility if we use it in other repositories with different name schemes.  

> This means we don't need to go back and change every branch of every repository to enable this, which is going to be dozens of PRs if we need to.

My original point was: things have been working on all flavours (software + branches) until now so let's try to keep the current code. For new branches (specially Dome ones) we will need this to work with nightlies.

Reply:
>  let's try to keep the current code

Yup, I appreciate the sentiment. I just worry about adding `gzdev_project_name` only where we need now, and in the future taking time to debug and realize we need in more places. Using `GITHUB_REPOSITORY` we should really keep the current code.

> Thought on sniffing the repo name to infer the name but I think that it would rest some flexibility if we use it in other repositories with different name schemes.

Gotcha. The only repo with a different naming scheme that I can think of is `sdformat`, and I think we could make that work if we need to. I think the main advantage is having to do less configuration for each package.

I just saw your reply. I had started some testing on branch [chapulina/gzdev](https://github.com/ignition-tooling/ubuntu-bionic-ci-action/compare/gzdev...chapulina/gzdev) and checked that the `${GITHUB_REPOSITORY}` variable is available inside `entrypoint.sh`. Let me know what you think.

If we do go with the `gzdev_project_name` variable, I'd suggest we call it just `project_name` so it can be used for other things too.

Reply:
> I just saw your reply. I had started some testing on branch [chapulina/gzdev](https://github.com/ignition-tooling/ubuntu-bionic-ci-action/compare/gzdev...chapulina/gzdev) and checked that the `${GITHUB_REPOSITORY}` variable is available inside `entrypoint.sh`. Let me know what you think. [Here's a successful run](https://github.com/ignitionrobotics/ign-transport/runs/892218353).

The approach of prioritize the reduced number of code paths and parameters is fine for me. I just pull your branch and clean a couple of echos. Did not touch any of the grouping changes although I made some comments in #15 .